tpmutil/structures.go

```diff
@@ -51,10 +141,31 @@ type responseHeader struct {
 // A Handle is a reference to a TPM object.
 type Handle uint32
 
+type handleList []Handle
+
+func (l *handleList) TPMMarshal(out io.Writer) error {
+	return fmt.Errorf("TPM should never marshal []Handle")
```

Comment:
make this "TPMMarhsal on []Handle is not supported yet".
no reason we couldn't implement this later

Reply:
Fair, it's just afaik none of the TPM commands require []Handle, hence why I had that comment.
Fixed.